docs/OMF.rst

```diff
@@ -56,6 +56,7 @@ Select PI Web API from the Endpoint options.
    - **Number Format:** Used to match Fledge data types to the data type configured in PI. The default is float64 but may be set to any OMF datatype that supports floating point values.
    - **Compression:** Compress the readings data before sending them to the PI Web API OMF endpoint.
      This setting is not related to data compression in the PI Data Archive.
+   - **Complex Types:** Used to force the plugin to send OMF data types as complex types rather than the newer linked types. Linked types are the default way to send data and allows assets to have different sets of data points in different readings. See :ref:`Linked_Types`.
```

Comment:
As there is new config item added to OMF plugin. Are we looking to update the screenshots for the OMF plugin info?

Reply:
I was delaying this until we have FOGL-7126 in place as this one is now so long it is hard to get a screen dump and it will change when 7126 is done.